.storybook/CustomPageTemplate.mdx

```diff
@@ -0,0 +1,22 @@
+import { Title, Description, Primary, Stories, ArgsTable, PRIMARY_STORY } from '@storybook/addon-docs';
+
+### Before creating a story, take time to read the instructions from 'Introduction To Creating Stories' for better documentation
```

Comment:
Minor; body text instead of heading 3 ?

Suggestion; can we add a link to the instructions page?